libraries/core_libs/storage/include/storage/storage.hpp

```diff
@@ -94,7 +94,7 @@ class DbStorage : public std::enable_shared_from_this<DbStorage> {
     // do not change/move
     COLUMN(default_column);
     // Contains full data for an executed PBFT block including PBFT block, cert votes, dag blocks and transactions
-    COLUMN(period_data);
+    COLUMN_W_COMP(period_data, getIntComparator<uint64_t>());
```

Comment:
I guess the COLUMN_W_COMP will do saving in order in DB. Am i right?

Reply:
Yes. This is needed for DeleteRange method in the db to work correctly.